src/main/java/com/echobox/api/linkedin/client/DefaultWebRequestor.java

```diff
@@ -68,6 +75,13 @@
 public class DefaultWebRequestor implements WebRequestor {
 
   private static final Logger LOGGER = LoggerFactory.getLogger(DefaultWebRequestor.class);
+  
+  private static final int DEFAULT_CONNECT_TIMEOUT_MS = 10000;
+  private static final int DEFAULT_READ_TIMEOUT_MS = 30000;
```

Comment:
These are the actual values used in [main](https://github.com/ebx/main/blob/dev/Lib/src/main/java/sh/ebx/api/social/linkedin/LinkedInAccountAPIManager.java#L175) by overriding the `customizeConnection` and what we went with in [tiktok](https://github.com/ebx/ebx-tiktok-sdk/commit/a9d37d3080f10a4b4ca1bc84622917708f3717d8#), so it makes sense to use as the defaults.